Dependencies/xr/Source/ARCore/XR.cpp

```diff
@@ -542,6 +536,43 @@ namespace xr
 
             ArCamera_release(camera);
 
+            // Suppress rendering if the camera did not produce the first frame yet.
+            // This is to avoid drawing possible leftover data from previous sessions if
+            // the texture is reused.
+            int64_t frameTimestamp{};
```

Comment:
I think maybe it makes sense to have this comment instead say something like: 
"Draw the camera texture to the Babylon render texture, suppress rendering if the camera did not produce the first frame yet."

That way you can easily tell at a glance what this chunk of code is responsible for before getting into the scope, since this chunk and the below chunk have the same comment it doesn't become apparent until you start reading the code inside the if statement.